CONTRIBUTING.md

```diff
@@ -1,3 +1,17 @@
 # Contributing
 
-Please fork and make a PR against `develop`. We love bugfixes and new features. Please open an issue or email semgrep@r2c.dev for help.
+## Upgrading to a new version of Bento
+
+```pipenv update```
+
+will update the version of Bento being used.
```

Comment:
```suggestion
will update the version of Semgrep being used.
```

Reply:
These are actually correct. It's in the contributing docs (for ourselves) on how to update the bento-cli we're using to run semgrep.

Reply:
👍